my-sms/app/assets/javascripts/institutes.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://jashkenas.github.com/coffee-script/
```

Comment:
File can be removed

Reply:
Fixed in 67792896a31da920aff3f1f48a47966c3180d376